## Configuration

In [6]:
import urllib.request
import requests
import json
import pandas as pd
import time
import os
import ast

In [34]:
mood = pd.read_csv('mood_ver3.csv')

print(mood.shape)
mood.head(10)

(178, 3)


Mood   data Unnamed: 2
0  interested     1        NaN
1       alert     1        NaN
2     excited     1        NaN
3    inspired     1        NaN
4       proud     1        NaN
5   confident     0        NaN
6  determined     1        NaN
7      strong     1        NaN
8     healthy     1        NaN
9      active     1        NaN

In [41]:
api_key = 'l8PvK_dxMZl0OhJ_2_rfBJDi-rhH_zjcrLTEVrs6xFE'
per_page = 30 # maximum 30

for keyword in mood['Mood '][163:170]: # ~ 168
    df = pd.DataFrame(columns=['color', 'description', 'alt_description', 'image', 'link', 'categories', 'tags', 'mood'])

    print(f'[{keyword}] Downloading..')
    
    page = 1
    while True:
        host = 'https://api.unsplash.com/search/photos?'
        parameters = f'client_id={api_key}&query={keyword}&page={page}&per_page={per_page}'

        req = requests.get(host+parameters)
        result = json.loads(req.text)
        
        if page == 1:
            print(' total page :', result['total_pages'])
            print(' total :', result['total'])
            
        print( page, end = ' ')
        
        for i in range(len(result['results'])):
            i_tmp = []
            i_tmp.append(result['results'][i]['color'])
            i_tmp.append(result['results'][i]['description'])
            i_tmp.append(result['results'][i]['alt_description'])
            i_tmp.append(result['results'][i]['urls']['raw'])
            i_tmp.append(result['results'][i]['links']['html'])
            i_tmp.append(result['results'][i]['categories'])
            i_tmp.append(result['results'][i]['tags'])
            i_tmp.append(keyword)

            df.loc[result['results'][i]['id']] = i_tmp

        time.sleep(1.5*60)
        page += 1
        if result['total_pages'] < page:
            print(f'\n{keyword} End.. \n')
            break

    if result['total_pages'] > 0:
        df.to_csv(f'./mood_metadata/mood_{keyword}.csv')
    else:
        df.to_csv(f'./mood_metadata/mood_{keyword}_None.csv')

[like being with others] Downloading..
 total page : 85
 total : 2535
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 
like being with others End.. 

[like having fun] Downloading..
 total page : 178
 total : 5320
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 

In [10]:
mood_ver4 = pd.read_csv('mood_ver4.csv')

print(mood_ver4.shape)
mood_ver4.head()

(181, 2)


mood Unnamed: 1
0     activated        NaN
1        active        NaN
2  affectionate        NaN
3        afraid        NaN
4         alert        NaN

In [18]:
mood_csv = {}
for file in os.listdir('./mood_metadata/'):
    if file.split('_')[0] == 'mood':
        tmp = pd.read_csv('./mood_metadata/'+file)
        mood_csv[file.split('_')[1]] = len(tmp)

In [44]:
tmp = sorted(mood_csv.items(), key=lambda x: x[1], reverse=True)

In [46]:
tmp_df = pd.DataFrame(tmp)

In [47]:
tmp_df.to_csv('mood_count.csv')

In [19]:
mood_csv

{'contemptuous': 0,
 'unhappy.csv': 85,
 'still.csv': 2316,
 'placid.csv': 38,
 'leisurely.csv': 4,
 'defiant.csv': 4,
 'like being alone.csv': 4604,
 'affectionate.csv': 20,
 'insecure.csv': 7,
 'bored.csv': 99,
 'deflant': 0,
 'speedy.csv': 11,
 'comfortable.csv': 156,
 'blameworthy': 0,
 'contented.csv': 3,
 'tired.csv': 328,
 'pleased.csv': 5,
 'lighthearted': 0,
 'strong.csv': 957,
 'lonely.csv': 1751,
 'witty.csv': 3,
 'bashful.csv': 3,
 'angry.csv': 289,
 'quick.csv': 110,
 'kindly': 0,
 'refreshed': 0,
 'uncertain.csv': 19,
 'energetic.csv': 87,
 'disgusted with self.csv': 2232,
 'like sharing with others.csv': 2396,
 'sleepy.csv': 301,
 'cautious.csv': 9,
 'clutched up.csv': 7767,
 'shocked.csv': 11,
 'aroused': 0,
 'ashamed.csv': 7,
 'worried.csv': 28,
 'angry at self.csv': 3008,
 'inadequate': 0,
 'cheerful.csv': 315,
 'contemplative.csv': 52,
 'full of energy.csv': 9061,
 'regretful': 0,
 'hopeless.csv': 68,
 'efficient.csv': 35,
 'active.csv': 533,
 'downhearted': 0,
 'fea

In [42]:
for file1 in os.listdir('./mood_metadata/'):
    print(f'\n\n{file1}과 비교중 ...')
    df1 = pd.read_csv('./mood_metadata/'+file1)
    for file2 in os.listdir('./mood_metadata/'):
        df2 = pd.read_csv('./mood_metadata/'+file2)
        df_sum = pd.concat([df1, df2])
        
        df1_len = len(df1)
        df2_len = len(df2)
        df_sum_nunique = df_sum['Unnamed: 0'].nunique()
        
        if (df1_len + df2_len) != df_sum_nunique:
            if file1 == file2:
                continue
            print(f'{file1} : {df1_len}, {file2} : {df2_len}')
            print(f'{df1_len+df2_len}, {df_sum_nunique}, 중복 : {df1_len+df2_len-df_sum_nunique}')

mood_contemptuous_None.csv과 비교중 ...
mood_unhappy.csv과 비교중 ...
mood_unhappy.csv : 85, mood_like being alone.csv : 4604
4689, 4674, 중복 : 15
mood_unhappy.csv : 85, mood_bored.csv : 99
184, 179, 중복 : 5
mood_unhappy.csv : 85, mood_tired.csv : 328
413, 407, 중복 : 6
mood_unhappy.csv : 85, mood_lonely.csv : 1751
1836, 1825, 중복 : 11
mood_unhappy.csv : 85, mood_angry.csv : 289
374, 368, 중복 : 6
mood_unhappy.csv : 85, mood_like sharing with others.csv : 2396
2481, 2480, 중복 : 1
mood_unhappy.csv : 85, mood_sleepy.csv : 301
386, 384, 중복 : 2
mood_unhappy.csv : 85, mood_clutched up.csv : 7767
7852, 7846, 중복 : 6
mood_unhappy.csv : 85, mood_ashamed.csv : 7
92, 91, 중복 : 1
mood_unhappy.csv : 85, mood_worried.csv : 28
113, 108, 중복 : 5
mood_unhappy.csv : 85, mood_angry at self.csv : 3008
3093, 3088, 중복 : 5
mood_unhappy.csv : 85, mood_cheerful.csv : 315
400, 399, 중복 : 1
mood_unhappy.csv : 85, mood_full of energy.csv : 9061
9146, 9142, 중복 : 4
mood_unhappy.csv : 85, mood_depressed.csv : 190
275, 267, 중복 : 8
mood

mood_placid.csv : 38, mood_clutched-up.csv : 8162
8200, 8199, 중복 : 1
mood_placid.csv : 38, mood_wide awake.csv : 1126
1164, 1163, 중복 : 1
mood_placid.csv : 38, mood_relaxed.csv : 345
383, 382, 중복 : 1
mood_placid.csv : 38, mood_quiet.csv : 1222
1260, 1259, 중복 : 1
mood_placid.csv : 38, mood_stirred-up.csv : 8044
8082, 8081, 중복 : 1
mood_placid.csv : 38, mood_self-centered.csv : 899
937, 936, 중복 : 1
mood_placid.csv : 38, mood_like being nice to others.csv : 3695
3733, 3732, 중복 : 1
mood_leisurely.csv과 비교중 ...
mood_leisurely.csv : 4, mood_like being alone.csv : 4604
4608, 4607, 중복 : 1
mood_leisurely.csv : 4, mood_strong.csv : 957
961, 960, 중복 : 1
mood_leisurely.csv : 4, mood_angry.csv : 289
293, 292, 중복 : 1
mood_leisurely.csv : 4, mood_quick.csv : 110
114, 113, 중복 : 1
mood_leisurely.csv : 4, mood_disgusted with self.csv : 2232
2236, 2234, 중복 : 2
mood_leisurely.csv : 4, mood_like sharing with others.csv : 2396
2400, 2399, 중복 : 1
mood_leisurely.csv : 4, mood_angry at self.csv : 3008
3012, 3010,

mood_like being alone.csv : 4604, mood_alone.csv : 3595
8199, 4983, 중복 : 3216
mood_like being alone.csv : 4604, mood_delighted.csv : 1
4605, 4604, 중복 : 1
mood_like being alone.csv : 4604, mood_amazed.csv : 48
4652, 4649, 중복 : 3
mood_like being alone.csv : 4604, mood_irritated.csv : 4
4608, 4607, 중복 : 1
mood_like being alone.csv : 4604, mood_scared.csv : 85
4689, 4677, 중복 : 12
mood_like being alone.csv : 4604, mood_upset.csv : 79
4683, 4673, 중복 : 10
mood_like being alone.csv : 4604, mood_shy.csv : 110
4714, 4709, 중복 : 5
mood_like being alone.csv : 4604, mood_mad.csv : 109
4713, 4705, 중복 : 8
mood_like being alone.csv : 4604, mood_stirred-up.csv : 8044
12648, 12544, 중복 : 104
mood_like being alone.csv : 4604, mood_self-centered.csv : 899
5503, 5461, 중복 : 42
mood_like being alone.csv : 4604, mood_rejected.csv : 5
4609, 4606, 중복 : 3
mood_like being alone.csv : 4604, mood_lazy.csv : 245
4849, 4846, 중복 : 3
mood_like being alone.csv : 4604, mood_sorry.csv : 29
4633, 4632, 중복 : 1
mood_like being

mood_comfortable.csv : 156, mood_active.csv : 533
689, 688, 중복 : 1
mood_comfortable.csv : 156, mood_at rest.csv : 2814
2970, 2960, 중복 : 10
mood_comfortable.csv : 156, mood_playful.csv : 309
465, 464, 중복 : 1
mood_comfortable.csv : 156, mood_safe.csv : 542
698, 697, 중복 : 1
mood_comfortable.csv : 156, mood_down.csv : 3110
3266, 3264, 중복 : 2
mood_comfortable.csv : 156, mood_distressed.csv : 87
243, 242, 중복 : 1
mood_comfortable.csv : 156, mood_healthy.csv : 2628
2784, 2778, 중복 : 6
mood_comfortable.csv : 156, mood_low.csv : 1293
1449, 1446, 중복 : 3
mood_comfortable.csv : 156, mood_frustrated.csv : 8
164, 163, 중복 : 1
mood_comfortable.csv : 156, mood_wild.csv : 7686
7842, 7841, 중복 : 1
mood_comfortable.csv : 156, mood_calm.csv : 3645
3801, 3798, 중복 : 3
mood_comfortable.csv : 156, mood_like having fun.csv : 4656
4812, 4805, 중복 : 7
mood_comfortable.csv : 156, mood_dissatisfied with self.csv : 2232
2388, 2387, 중복 : 1
mood_comfortable.csv : 156, mood_clutched-up.csv : 8162
8318, 8313, 중복 : 5
mood_co

mood_strong.csv : 957, mood_like sharing with others.csv : 2396
3353, 3346, 중복 : 7
mood_strong.csv : 957, mood_sleepy.csv : 301
1258, 1245, 중복 : 13
mood_strong.csv : 957, mood_clutched up.csv : 7767
8724, 8688, 중복 : 36
mood_strong.csv : 957, mood_worried.csv : 28
985, 984, 중복 : 1
mood_strong.csv : 957, mood_angry at self.csv : 3008
3965, 3931, 중복 : 34
mood_strong.csv : 957, mood_cheerful.csv : 315
1272, 1268, 중복 : 4
mood_strong.csv : 957, mood_full of energy.csv : 9061
10018, 9983, 중복 : 35
mood_strong.csv : 957, mood_hopeless.csv : 68
1025, 1023, 중복 : 2
mood_strong.csv : 957, mood_active.csv : 533
1490, 1465, 중복 : 25
mood_strong.csv : 957, mood_fearless.csv : 355
1312, 1062, 중복 : 250
mood_strong.csv : 957, mood_at rest.csv : 2814
3771, 3760, 중복 : 11
mood_strong.csv : 957, mood_depressed.csv : 190
1147, 1146, 중복 : 1
mood_strong.csv : 957, mood_safe.csv : 542
1499, 1495, 중복 : 4
mood_strong.csv : 957, mood_like everything is going wrong.csv : 1373
2330, 2321, 중복 : 9
mood_strong.csv : 957,

mood_lonely.csv : 1751, mood_stirred-up.csv : 8044
9795, 9766, 중복 : 29
mood_lonely.csv : 1751, mood_self-centered.csv : 899
2650, 2639, 중복 : 11
mood_lonely.csv : 1751, mood_rejected.csv : 5
1756, 1754, 중복 : 2
mood_lonely.csv : 1751, mood_lazy.csv : 245
1996, 1993, 중복 : 3
mood_lonely.csv : 1751, mood_sorry.csv : 29
1780, 1779, 중복 : 1
mood_lonely.csv : 1751, mood_engaged in thought.csv : 7026
8777, 8713, 중복 : 64
mood_lonely.csv : 1751, mood_sick.csv : 245
1996, 1990, 중복 : 6
mood_lonely.csv : 1751, mood_friendly.csv : 318
2069, 2066, 중복 : 3
mood_lonely.csv : 1751, mood_at ease.csv : 1799
3550, 3541, 중복 : 9
mood_lonely.csv : 1751, mood_bad-tempered.csv : 604
2355, 2348, 중복 : 7
mood_lonely.csv : 1751, mood_like being nice to others.csv : 3695
5446, 5423, 중복 : 23
mood_lonely.csv : 1751, mood_dull.csv : 69
1820, 1817, 중복 : 3
mood_lonely.csv : 1751, mood_super.csv : 815
2566, 2418, 중복 : 148
mood_lonely.csv : 1751, mood_sexy.csv : 936
2687, 2681, 중복 : 6
mood_lonely.csv : 1751, mood_nice.csv : 1

mood_quick.csv : 110, mood_happy.csv : 6059
6169, 6165, 중복 : 4
mood_quick.csv : 110, mood_sad.csv : 2086
2196, 2195, 중복 : 1
mood_quick.csv : 110, mood_quiet.csv : 1222
1332, 1331, 중복 : 1
mood_quick.csv : 110, mood_carefree.csv : 73
183, 182, 중복 : 1
mood_quick.csv : 110, mood_alone.csv : 3595
3705, 3704, 중복 : 1
mood_quick.csv : 110, mood_stirred-up.csv : 8044
8154, 8142, 중복 : 12
mood_quick.csv : 110, mood_lazy.csv : 245
355, 354, 중복 : 1
mood_quick.csv : 110, mood_engaged in thought.csv : 7026
7136, 7130, 중복 : 6
mood_quick.csv : 110, mood_at ease.csv : 1799
1909, 1907, 중복 : 2
mood_quick.csv : 110, mood_like being nice to others.csv : 3695
3805, 3801, 중복 : 4
mood_quick.csv : 110, mood_dull.csv : 69
179, 178, 중복 : 1
mood_quick.csv : 110, mood_super.csv : 815
925, 923, 중복 : 2
mood_quick.csv : 110, mood_alert.csv : 87
197, 196, 중복 : 1
mood_quick.csv : 110, mood_nice.csv : 1265
1375, 1374, 중복 : 1
mood_kindly_None.csv과 비교중 ...
mood_refreshed_None.csv과 비교중 ...
mood_uncertain.csv과 비교중 ...
mood_u

mood_disgusted with self.csv : 2232, mood_full-of-pep.csv : 8942
11174, 11157, 중복 : 17
mood_disgusted with self.csv : 2232, mood_dissatisfied with self.csv : 2232
4464, 2298, 중복 : 2166
mood_disgusted with self.csv : 2232, mood_excited.csv : 107
2339, 2338, 중복 : 1
mood_disgusted with self.csv : 2232, mood_anxious.csv : 46
2278, 2276, 중복 : 2
mood_disgusted with self.csv : 2232, mood_clutched-up.csv : 8162
10394, 10337, 중복 : 57
mood_disgusted with self.csv : 2232, mood_content.csv : 301
2533, 2528, 중복 : 5
mood_disgusted with self.csv : 2232, mood_like caring about others.csv : 820
3052, 3032, 중복 : 20
mood_disgusted with self.csv : 2232, mood_wide awake.csv : 1126
3358, 3351, 중복 : 7
mood_disgusted with self.csv : 2232, mood_annoyed.csv : 48
2280, 2279, 중복 : 1
mood_disgusted with self.csv : 2232, mood_glad.csv : 28
2260, 2259, 중복 : 1
mood_disgusted with self.csv : 2232, mood_happy.csv : 6059
8291, 8219, 중복 : 72
mood_disgusted with self.csv : 2232, mood_serious.csv : 395
2627, 2615, 중복 : 12


mood_like sharing with others.csv : 2396, mood_engaged in thought.csv : 7026
9422, 9088, 중복 : 334
mood_like sharing with others.csv : 2396, mood_loving.csv : 205
2601, 2593, 중복 : 8
mood_like sharing with others.csv : 2396, mood_sick.csv : 245
2641, 2637, 중복 : 4
mood_like sharing with others.csv : 2396, mood_friendly.csv : 318
2714, 2705, 중복 : 9
mood_like sharing with others.csv : 2396, mood_at ease.csv : 1799
4195, 4154, 중복 : 41
mood_like sharing with others.csv : 2396, mood_bad-tempered.csv : 604
3000, 2997, 중복 : 3
mood_like sharing with others.csv : 2396, mood_like being nice to others.csv : 3695
6091, 5343, 중복 : 748
mood_like sharing with others.csv : 2396, mood_super.csv : 815
3211, 3201, 중복 : 10
mood_like sharing with others.csv : 2396, mood_sexy.csv : 936
3332, 3319, 중복 : 13
mood_like sharing with others.csv : 2396, mood_guilty.csv : 10
2406, 2405, 중복 : 1
mood_like sharing with others.csv : 2396, mood_nice.csv : 1265
3661, 3648, 중복 : 13
mood_sleepy.csv과 비교중 ...
mood_sleepy.csv : 

mood_clutched up.csv : 7767, mood_like being with others.csv : 2221
9988, 9933, 중복 : 55
mood_clutched up.csv : 7767, mood_healthy.csv : 2628
10395, 10318, 중복 : 77
mood_clutched up.csv : 7767, mood_sociable.csv : 1
7768, 7767, 중복 : 1
mood_clutched up.csv : 7767, mood_low.csv : 1293
9060, 8996, 중복 : 64
mood_clutched up.csv : 7767, mood_lively.csv : 36
7803, 7800, 중복 : 3
mood_clutched up.csv : 7767, mood_frustrated.csv : 8
7775, 7774, 중복 : 1
mood_clutched up.csv : 7767, mood_wild.csv : 7686
15453, 15226, 중복 : 227
mood_clutched up.csv : 7767, mood_intense.csv : 259
8026, 8013, 중복 : 13
mood_clutched up.csv : 7767, mood_calm.csv : 3645
11412, 11355, 중복 : 57
mood_clutched up.csv : 7767, mood_concentrating.csv : 5
7772, 7771, 중복 : 1
mood_clutched up.csv : 7767, mood_like having fun.csv : 4656
12423, 12330, 중복 : 93
mood_clutched up.csv : 7767, mood_full-of-pep.csv : 8942
16709, 16617, 중복 : 92
mood_clutched up.csv : 7767, mood_dissatisfied with self.csv : 2232
9999, 9937, 중복 : 62
mood_clutched u

mood_worried.csv : 28, mood_down.csv : 3110
3138, 3133, 중복 : 5
mood_worried.csv : 28, mood_healthy.csv : 2628
2656, 2652, 중복 : 4
mood_worried.csv : 28, mood_frustrated.csv : 8
36, 35, 중복 : 1
mood_worried.csv : 28, mood_calm.csv : 3645
3673, 3672, 중복 : 1
mood_worried.csv : 28, mood_like having fun.csv : 4656
4684, 4682, 중복 : 2
mood_worried.csv : 28, mood_full-of-pep.csv : 8942
8970, 8969, 중복 : 1
mood_worried.csv : 28, mood_excited.csv : 107
135, 134, 중복 : 1
mood_worried.csv : 28, mood_nervous.csv : 38
66, 65, 중복 : 1
mood_worried.csv : 28, mood_anxious.csv : 46
74, 72, 중복 : 2
mood_worried.csv : 28, mood_clutched-up.csv : 8162
8190, 8188, 중복 : 2
mood_worried.csv : 28, mood_content.csv : 301
329, 328, 중복 : 1
mood_worried.csv : 28, mood_like caring about others.csv : 820
848, 847, 중복 : 1
mood_worried.csv : 28, mood_wide awake.csv : 1126
1154, 1152, 중복 : 2
mood_worried.csv : 28, mood_happy.csv : 6059
6087, 6083, 중복 : 4
mood_worried.csv : 28, mood_serious.csv : 395
423, 421, 중복 : 2
mood_worri

mood_cheerful.csv과 비교중 ...
mood_cheerful.csv : 315, mood_unhappy.csv : 85
400, 399, 중복 : 1
mood_cheerful.csv : 315, mood_still.csv : 2316
2631, 2630, 중복 : 1
mood_cheerful.csv : 315, mood_leisurely.csv : 4
319, 318, 중복 : 1
mood_cheerful.csv : 315, mood_like being alone.csv : 4604
4919, 4912, 중복 : 7
mood_cheerful.csv : 315, mood_affectionate.csv : 20
335, 329, 중복 : 6
mood_cheerful.csv : 315, mood_bored.csv : 99
414, 413, 중복 : 1
mood_cheerful.csv : 315, mood_comfortable.csv : 156
471, 466, 중복 : 5
mood_cheerful.csv : 315, mood_pleased.csv : 5
320, 319, 중복 : 1
mood_cheerful.csv : 315, mood_strong.csv : 957
1272, 1268, 중복 : 4
mood_cheerful.csv : 315, mood_angry.csv : 289
604, 602, 중복 : 2
mood_cheerful.csv : 315, mood_quick.csv : 110
425, 423, 중복 : 2
mood_cheerful.csv : 315, mood_energetic.csv : 87
402, 401, 중복 : 1
mood_cheerful.csv : 315, mood_disgusted with self.csv : 2232
2547, 2540, 중복 : 7
mood_cheerful.csv : 315, mood_like sharing with others.csv : 2396
2711, 2703, 중복 : 8
mood_cheerful.c

mood_full of energy.csv : 9061, mood_healthy.csv : 2628
11689, 11620, 중복 : 69
mood_full of energy.csv : 9061, mood_low.csv : 1293
10354, 10333, 중복 : 21
mood_full of energy.csv : 9061, mood_lively.csv : 36
9097, 9096, 중복 : 1
mood_full of energy.csv : 9061, mood_wild.csv : 7686
16747, 16699, 중복 : 48
mood_full of energy.csv : 9061, mood_intense.csv : 259
9320, 9309, 중복 : 11
mood_full of energy.csv : 9061, mood_calm.csv : 3645
12706, 12649, 중복 : 57
mood_full of energy.csv : 9061, mood_like having fun.csv : 4656
13717, 13650, 중복 : 67
mood_full of energy.csv : 9061, mood_forceful.csv : 1
9062, 9061, 중복 : 1
mood_full of energy.csv : 9061, mood_full-of-pep.csv : 8942
18003, 10665, 중복 : 7338
mood_full of energy.csv : 9061, mood_dissatisfied with self.csv : 2232
11293, 11270, 중복 : 23
mood_full of energy.csv : 9061, mood_excited.csv : 107
9168, 9165, 중복 : 3
mood_full of energy.csv : 9061, mood_nervous.csv : 38
9099, 9098, 중복 : 1
mood_full of energy.csv : 9061, mood_anxious.csv : 46
9107, 9102, 중복

mood_efficient.csv : 35, mood_full-of-pep.csv : 8942
8977, 8976, 중복 : 1
mood_efficient.csv : 35, mood_dissatisfied with self.csv : 2232
2267, 2266, 중복 : 1
mood_efficient.csv : 35, mood_nervous.csv : 38
73, 72, 중복 : 1
mood_efficient.csv : 35, mood_clutched-up.csv : 8162
8197, 8196, 중복 : 1
mood_efficient.csv : 35, mood_content.csv : 301
336, 328, 중복 : 8
mood_efficient.csv : 35, mood_like caring about others.csv : 820
855, 852, 중복 : 3
mood_efficient.csv : 35, mood_wide awake.csv : 1126
1161, 1160, 중복 : 1
mood_efficient.csv : 35, mood_happy.csv : 6059
6094, 6092, 중복 : 2
mood_efficient.csv : 35, mood_relaxed.csv : 345
380, 378, 중복 : 2
mood_efficient.csv : 35, mood_stirred-up.csv : 8044
8079, 8078, 중복 : 1
mood_efficient.csv : 35, mood_self-centered.csv : 899
934, 933, 중복 : 1
mood_efficient.csv : 35, mood_engaged in thought.csv : 7026
7061, 7051, 중복 : 10
mood_efficient.csv : 35, mood_at ease.csv : 1799
1834, 1833, 중복 : 1
mood_efficient.csv : 35, mood_like being nice to others.csv : 3695
3730,

mood_at rest.csv : 2814, mood_strong.csv : 957
3771, 3760, 중복 : 11
mood_at rest.csv : 2814, mood_lonely.csv : 1751
4565, 4534, 중복 : 31
mood_at rest.csv : 2814, mood_bashful.csv : 3
2817, 2816, 중복 : 1
mood_at rest.csv : 2814, mood_angry.csv : 289
3103, 3102, 중복 : 1
mood_at rest.csv : 2814, mood_quick.csv : 110
2924, 2922, 중복 : 2
mood_at rest.csv : 2814, mood_disgusted with self.csv : 2232
5046, 4972, 중복 : 74
mood_at rest.csv : 2814, mood_like sharing with others.csv : 2396
5210, 5164, 중복 : 46
mood_at rest.csv : 2814, mood_sleepy.csv : 301
3115, 3094, 중복 : 21
mood_at rest.csv : 2814, mood_clutched up.csv : 7767
10581, 10500, 중복 : 81
mood_at rest.csv : 2814, mood_worried.csv : 28
2842, 2840, 중복 : 2
mood_at rest.csv : 2814, mood_angry at self.csv : 3008
5822, 4008, 중복 : 1814
mood_at rest.csv : 2814, mood_cheerful.csv : 315
3129, 3118, 중복 : 11
mood_at rest.csv : 2814, mood_contemplative.csv : 52
2866, 2865, 중복 : 1
mood_at rest.csv : 2814, mood_full of energy.csv : 9061
11875, 11821, 중복 : 54

mood_playful.csv : 309, mood_stirred-up.csv : 8044
8353, 8334, 중복 : 19
mood_playful.csv : 309, mood_engaged in thought.csv : 7026
7335, 7316, 중복 : 19
mood_playful.csv : 309, mood_loving.csv : 205
514, 513, 중복 : 1
mood_playful.csv : 309, mood_friendly.csv : 318
627, 604, 중복 : 23
mood_playful.csv : 309, mood_at ease.csv : 1799
2108, 2104, 중복 : 4
mood_playful.csv : 309, mood_bad-tempered.csv : 604
913, 909, 중복 : 4
mood_playful.csv : 309, mood_like being nice to others.csv : 3695
4004, 3999, 중복 : 5
mood_playful.csv : 309, mood_sexy.csv : 936
1245, 1227, 중복 : 18
mood_playful.csv : 309, mood_alert.csv : 87
396, 395, 중복 : 1
mood_dubious_None.csv과 비교중 ...
mood_startled.csv과 비교중 ...
mood_startled.csv : 5, mood_shocked.csv : 11
16, 15, 중복 : 1
mood_startled.csv : 5, mood_surprised.csv : 39
44, 42, 중복 : 2
mood_startled.csv : 5, mood_wide awake.csv : 1126
1131, 1130, 중복 : 1
mood_startled.csv : 5, mood_scared.csv : 85
90, 89, 중복 : 1
mood_droopy.csv과 비교중 ...
mood_droopy.csv : 4, mood_still.csv : 2316

mood_safe.csv : 542, mood_like sharing with others.csv : 2396
2938, 2934, 중복 : 4
mood_safe.csv : 542, mood_cautious.csv : 9
551, 550, 중복 : 1
mood_safe.csv : 542, mood_clutched up.csv : 7767
8309, 8302, 중복 : 7
mood_safe.csv : 542, mood_angry at self.csv : 3008
3550, 3521, 중복 : 29
mood_safe.csv : 542, mood_full of energy.csv : 9061
9603, 9597, 중복 : 6
mood_safe.csv : 542, mood_at rest.csv : 2814
3356, 3336, 중복 : 20
mood_safe.csv : 542, mood_playful.csv : 309
851, 848, 중복 : 3
mood_safe.csv : 542, mood_like everything is going wrong.csv : 1373
1915, 1911, 중복 : 4
mood_safe.csv : 542, mood_down.csv : 3110
3652, 3641, 중복 : 11
mood_safe.csv : 542, mood_like being with others.csv : 2221
2763, 2759, 중복 : 4
mood_safe.csv : 542, mood_healthy.csv : 2628
3170, 3158, 중복 : 12
mood_safe.csv : 542, mood_low.csv : 1293
1835, 1834, 중복 : 1
mood_safe.csv : 542, mood_wild.csv : 7686
8228, 8223, 중복 : 5
mood_safe.csv : 542, mood_intense.csv : 259
801, 800, 중복 : 1
mood_safe.csv : 542, mood_calm.csv : 3645
4187, 

mood_like everything is going wrong.csv : 1373, mood_loving.csv : 205
1578, 1570, 중복 : 8
mood_like everything is going wrong.csv : 1373, mood_sick.csv : 245
1618, 1616, 중복 : 2
mood_like everything is going wrong.csv : 1373, mood_friendly.csv : 318
1691, 1686, 중복 : 5
mood_like everything is going wrong.csv : 1373, mood_at ease.csv : 1799
3172, 3150, 중복 : 22
mood_like everything is going wrong.csv : 1373, mood_bad-tempered.csv : 604
1977, 1973, 중복 : 4
mood_like everything is going wrong.csv : 1373, mood_like being nice to others.csv : 3695
5068, 4430, 중복 : 638
mood_like everything is going wrong.csv : 1373, mood_super.csv : 815
2188, 2185, 중복 : 3
mood_like everything is going wrong.csv : 1373, mood_sexy.csv : 936
2309, 2305, 중복 : 4
mood_like everything is going wrong.csv : 1373, mood_nice.csv : 1265
2638, 2625, 중복 : 13
mood_bugged_None.csv과 비교중 ...
mood_scornful_None.csv과 비교중 ...
mood_afraid.csv과 비교중 ...
mood_afraid.csv : 22, mood_like being alone.csv : 4604
4626, 4622, 중복 : 4
mood_afrai

mood_down.csv : 3110, mood_serious.csv : 395
3505, 3498, 중복 : 7
mood_down.csv : 3110, mood_sad.csv : 2086
5196, 5111, 중복 : 85
mood_down.csv : 3110, mood_relaxed.csv : 345
3455, 3437, 중복 : 18
mood_down.csv : 3110, mood_quiet.csv : 1222
4332, 4320, 중복 : 12
mood_down.csv : 3110, mood_attentive.csv : 14
3124, 3123, 중복 : 1
mood_down.csv : 3110, mood_carefree.csv : 73
3183, 3180, 중복 : 3
mood_down.csv : 3110, mood_mean.csv : 69
3179, 3178, 중복 : 1
mood_down.csv : 3110, mood_alone.csv : 3595
6705, 6645, 중복 : 60
mood_down.csv : 3110, mood_scared.csv : 85
3195, 3194, 중복 : 1
mood_down.csv : 3110, mood_subdued.csv : 9
3119, 3118, 중복 : 1
mood_down.csv : 3110, mood_upset.csv : 79
3189, 3182, 중복 : 7
mood_down.csv : 3110, mood_shy.csv : 110
3220, 3214, 중복 : 6
mood_down.csv : 3110, mood_mad.csv : 109
3219, 3218, 중복 : 1
mood_down.csv : 3110, mood_stirred-up.csv : 8044
11154, 11003, 중복 : 151
mood_down.csv : 3110, mood_self-centered.csv : 899
4009, 3988, 중복 : 21
mood_down.csv : 3110, mood_rejected.csv : 5


mood_like being with others.csv : 2221, mood_angry at self.csv : 3008
5229, 5167, 중복 : 62
mood_like being with others.csv : 2221, mood_cheerful.csv : 315
2536, 2528, 중복 : 8
mood_like being with others.csv : 2221, mood_full of energy.csv : 9061
11282, 11259, 중복 : 23
mood_like being with others.csv : 2221, mood_efficient.csv : 35
2256, 2253, 중복 : 3
mood_like being with others.csv : 2221, mood_active.csv : 533
2754, 2751, 중복 : 3
mood_like being with others.csv : 2221, mood_fearless.csv : 355
2576, 2573, 중복 : 3
mood_like being with others.csv : 2221, mood_at rest.csv : 2814
5035, 4997, 중복 : 38
mood_like being with others.csv : 2221, mood_playful.csv : 309
2530, 2525, 중복 : 5
mood_like being with others.csv : 2221, mood_depressed.csv : 190
2411, 2409, 중복 : 2
mood_like being with others.csv : 2221, mood_safe.csv : 542
2763, 2759, 중복 : 4
mood_like being with others.csv : 2221, mood_like everything is going wrong.csv : 1373
3594, 3020, 중복 : 574
mood_like being with others.csv : 2221, mood_afrai

mood_healthy.csv : 2628, mood_alone.csv : 3595
6223, 6201, 중복 : 22
mood_healthy.csv : 2628, mood_scared.csv : 85
2713, 2712, 중복 : 1
mood_healthy.csv : 2628, mood_upset.csv : 79
2707, 2704, 중복 : 3
mood_healthy.csv : 2628, mood_stirred-up.csv : 8044
10672, 10589, 중복 : 83
mood_healthy.csv : 2628, mood_self-centered.csv : 899
3527, 3508, 중복 : 19
mood_healthy.csv : 2628, mood_lazy.csv : 245
2873, 2868, 중복 : 5
mood_healthy.csv : 2628, mood_engaged in thought.csv : 7026
9654, 9622, 중복 : 32
mood_healthy.csv : 2628, mood_loving.csv : 205
2833, 2832, 중복 : 1
mood_healthy.csv : 2628, mood_sick.csv : 245
2873, 2869, 중복 : 4
mood_healthy.csv : 2628, mood_friendly.csv : 318
2946, 2930, 중복 : 16
mood_healthy.csv : 2628, mood_at ease.csv : 1799
4427, 4374, 중복 : 53
mood_healthy.csv : 2628, mood_bad-tempered.csv : 604
3232, 3228, 중복 : 4
mood_healthy.csv : 2628, mood_like being nice to others.csv : 3695
6323, 6288, 중복 : 35
mood_healthy.csv : 2628, mood_super.csv : 815
3443, 3438, 중복 : 5
mood_healthy.csv : 2

mood_frustrated.csv : 8, mood_annoyed.csv : 48
56, 54, 중복 : 2
mood_frustrated.csv : 8, mood_happy.csv : 6059
6067, 6065, 중복 : 2
mood_frustrated.csv : 8, mood_serious.csv : 395
403, 402, 중복 : 1
mood_frustrated.csv : 8, mood_sad.csv : 2086
2094, 2092, 중복 : 2
mood_frustrated.csv : 8, mood_quiet.csv : 1222
1230, 1229, 중복 : 1
mood_frustrated.csv : 8, mood_determined.csv : 27
35, 34, 중복 : 1
mood_frustrated.csv : 8, mood_alone.csv : 3595
3603, 3602, 중복 : 1
mood_frustrated.csv : 8, mood_irritated.csv : 4
12, 11, 중복 : 1
mood_frustrated.csv : 8, mood_upset.csv : 79
87, 85, 중복 : 2
mood_frustrated.csv : 8, mood_mad.csv : 109
117, 116, 중복 : 1
mood_frustrated.csv : 8, mood_stirred-up.csv : 8044
8052, 8051, 중복 : 1
mood_frustrated.csv : 8, mood_self-centered.csv : 899
907, 906, 중복 : 1
mood_frustrated.csv : 8, mood_engaged in thought.csv : 7026
7034, 7033, 중복 : 1
mood_frustrated.csv : 8, mood_at ease.csv : 1799
1807, 1806, 중복 : 1
mood_wild.csv과 비교중 ...
mood_wild.csv : 7686, mood_unhappy.csv : 85
7771, 

mood_intense.csv : 259, mood_down.csv : 3110
3369, 3364, 중복 : 5
mood_intense.csv : 259, mood_enthusiastic.csv : 18
277, 276, 중복 : 1
mood_intense.csv : 259, mood_like being with others.csv : 2221
2480, 2478, 중복 : 2
mood_intense.csv : 259, mood_healthy.csv : 2628
2887, 2884, 중복 : 3
mood_intense.csv : 259, mood_low.csv : 1293
1552, 1549, 중복 : 3
mood_intense.csv : 259, mood_wild.csv : 7686
7945, 7935, 중복 : 10
mood_intense.csv : 259, mood_calm.csv : 3645
3904, 3903, 중복 : 1
mood_intense.csv : 259, mood_like having fun.csv : 4656
4915, 4909, 중복 : 6
mood_intense.csv : 259, mood_full-of-pep.csv : 8942
9201, 9193, 중복 : 8
mood_intense.csv : 259, mood_dissatisfied with self.csv : 2232
2491, 2490, 중복 : 1
mood_intense.csv : 259, mood_suspicious.csv : 12
271, 270, 중복 : 1
mood_intense.csv : 259, mood_excited.csv : 107
366, 365, 중복 : 1
mood_intense.csv : 259, mood_clutched-up.csv : 8162
8421, 8405, 중복 : 16
mood_intense.csv : 259, mood_like caring about others.csv : 820
1079, 1078, 중복 : 1
mood_intense.c

mood_concentrating.csv : 5, mood_serious.csv : 395
400, 399, 중복 : 1
mood_concentrating.csv : 5, mood_upset.csv : 79
84, 83, 중복 : 1
mood_concentrating.csv : 5, mood_stirred-up.csv : 8044
8049, 8048, 중복 : 1
mood_like having fun.csv과 비교중 ...
mood_like having fun.csv : 4656, mood_unhappy.csv : 85
4741, 4739, 중복 : 2
mood_like having fun.csv : 4656, mood_still.csv : 2316
6972, 6932, 중복 : 40
mood_like having fun.csv : 4656, mood_like being alone.csv : 4604
9260, 8591, 중복 : 669
mood_like having fun.csv : 4656, mood_affectionate.csv : 20
4676, 4675, 중복 : 1
mood_like having fun.csv : 4656, mood_bored.csv : 99
4755, 4752, 중복 : 3
mood_like having fun.csv : 4656, mood_comfortable.csv : 156
4812, 4805, 중복 : 7
mood_like having fun.csv : 4656, mood_tired.csv : 328
4984, 4980, 중복 : 4
mood_like having fun.csv : 4656, mood_pleased.csv : 5
4661, 4660, 중복 : 1
mood_like having fun.csv : 4656, mood_strong.csv : 957
5613, 5593, 중복 : 20
mood_like having fun.csv : 4656, mood_lonely.csv : 1751
6407, 6183, 중복 : 2

mood_full-of-pep.csv : 8942, mood_angry at self.csv : 3008
11950, 11902, 중복 : 48
mood_full-of-pep.csv : 8942, mood_cheerful.csv : 315
9257, 9254, 중복 : 3
mood_full-of-pep.csv : 8942, mood_contemplative.csv : 52
8994, 8993, 중복 : 1
mood_full-of-pep.csv : 8942, mood_full of energy.csv : 9061
18003, 10665, 중복 : 7338
mood_full-of-pep.csv : 8942, mood_hopeless.csv : 68
9010, 9006, 중복 : 4
mood_full-of-pep.csv : 8942, mood_efficient.csv : 35
8977, 8976, 중복 : 1
mood_full-of-pep.csv : 8942, mood_active.csv : 533
9475, 9464, 중복 : 11
mood_full-of-pep.csv : 8942, mood_fearless.csv : 355
9297, 9295, 중복 : 2
mood_full-of-pep.csv : 8942, mood_at rest.csv : 2814
11756, 11714, 중복 : 42
mood_full-of-pep.csv : 8942, mood_playful.csv : 309
9251, 9250, 중복 : 1
mood_full-of-pep.csv : 8942, mood_droopy.csv : 4
8946, 8945, 중복 : 1
mood_full-of-pep.csv : 8942, mood_depressed.csv : 190
9132, 9125, 중복 : 7
mood_full-of-pep.csv : 8942, mood_safe.csv : 542
9484, 9478, 중복 : 6
mood_full-of-pep.csv : 8942, mood_like everyth

mood_dissatisfied with self.csv : 2232, mood_like caring about others.csv : 820
3052, 3032, 중복 : 20
mood_dissatisfied with self.csv : 2232, mood_wide awake.csv : 1126
3358, 3351, 중복 : 7
mood_dissatisfied with self.csv : 2232, mood_annoyed.csv : 48
2280, 2279, 중복 : 1
mood_dissatisfied with self.csv : 2232, mood_glad.csv : 28
2260, 2259, 중복 : 1
mood_dissatisfied with self.csv : 2232, mood_happy.csv : 6059
8291, 8225, 중복 : 66
mood_dissatisfied with self.csv : 2232, mood_serious.csv : 395
2627, 2615, 중복 : 12
mood_dissatisfied with self.csv : 2232, mood_sad.csv : 2086
4318, 4297, 중복 : 21
mood_dissatisfied with self.csv : 2232, mood_relaxed.csv : 345
2577, 2572, 중복 : 5
mood_dissatisfied with self.csv : 2232, mood_joyful.csv : 167
2399, 2398, 중복 : 1
mood_dissatisfied with self.csv : 2232, mood_quiet.csv : 1222
3454, 3442, 중복 : 12
mood_dissatisfied with self.csv : 2232, mood_satisfied.csv : 13
2245, 2244, 중복 : 1
mood_dissatisfied with self.csv : 2232, mood_determined.csv : 27
2259, 2258, 중복 : 

mood_nervous.csv : 38, mood_efficient.csv : 35
73, 72, 중복 : 1
mood_nervous.csv : 38, mood_at rest.csv : 2814
2852, 2851, 중복 : 1
mood_nervous.csv : 38, mood_depressed.csv : 190
228, 227, 중복 : 1
mood_nervous.csv : 38, mood_afraid.csv : 22
60, 59, 중복 : 1
mood_nervous.csv : 38, mood_down.csv : 3110
3148, 3146, 중복 : 2
mood_nervous.csv : 38, mood_frustrated.csv : 8
46, 45, 중복 : 1
mood_nervous.csv : 38, mood_concentrating.csv : 5
43, 42, 중복 : 1
mood_nervous.csv : 38, mood_full-of-pep.csv : 8942
8980, 8979, 중복 : 1
mood_nervous.csv : 38, mood_frightened.csv : 8
46, 45, 중복 : 1
mood_nervous.csv : 38, mood_anxious.csv : 46
84, 74, 중복 : 10
mood_nervous.csv : 38, mood_clutched-up.csv : 8162
8200, 8197, 중복 : 3
mood_nervous.csv : 38, mood_annoyed.csv : 48
86, 85, 중복 : 1
mood_nervous.csv : 38, mood_happy.csv : 6059
6097, 6096, 중복 : 1
mood_nervous.csv : 38, mood_sad.csv : 2086
2124, 2120, 중복 : 4
mood_nervous.csv : 38, mood_alone.csv : 3595
3633, 3631, 중복 : 2
mood_nervous.csv : 38, mood_scared.csv : 85
1

mood_clutched-up.csv : 8162, mood_like having fun.csv : 4656
12818, 12712, 중복 : 106
mood_clutched-up.csv : 8162, mood_full-of-pep.csv : 8942
17104, 17013, 중복 : 91
mood_clutched-up.csv : 8162, mood_dissatisfied with self.csv : 2232
10394, 10335, 중복 : 59
mood_clutched-up.csv : 8162, mood_suspicious.csv : 12
8174, 8172, 중복 : 2
mood_clutched-up.csv : 8162, mood_excited.csv : 107
8269, 8262, 중복 : 7
mood_clutched-up.csv : 8162, mood_nervous.csv : 38
8200, 8197, 중복 : 3
mood_clutched-up.csv : 8162, mood_frightened.csv : 8
8170, 8169, 중복 : 1
mood_clutched-up.csv : 8162, mood_anxious.csv : 46
8208, 8205, 중복 : 3
mood_clutched-up.csv : 8162, mood_skeptical.csv : 1
8163, 8162, 중복 : 1
mood_clutched-up.csv : 8162, mood_content.csv : 301
8463, 8452, 중복 : 11
mood_clutched-up.csv : 8162, mood_like caring about others.csv : 820
8982, 8961, 중복 : 21
mood_clutched-up.csv : 8162, mood_wide awake.csv : 1126
9288, 9237, 중복 : 51
mood_clutched-up.csv : 8162, mood_tense.csv : 10
8172, 8171, 중복 : 1
mood_clutched-u

mood_content.csv : 301, mood_nice.csv : 1265
1566, 1564, 중복 : 2
mood_panicky_None.csv과 비교중 ...
mood_like caring about others.csv과 비교중 ...
mood_like caring about others.csv : 820, mood_still.csv : 2316
3136, 3128, 중복 : 8
mood_like caring about others.csv : 820, mood_like being alone.csv : 4604
5424, 4807, 중복 : 617
mood_like caring about others.csv : 820, mood_affectionate.csv : 20
840, 838, 중복 : 2
mood_like caring about others.csv : 820, mood_tired.csv : 328
1148, 1147, 중복 : 1
mood_like caring about others.csv : 820, mood_strong.csv : 957
1777, 1773, 중복 : 4
mood_like caring about others.csv : 820, mood_lonely.csv : 1751
2571, 2569, 중복 : 2
mood_like caring about others.csv : 820, mood_quick.csv : 110
930, 928, 중복 : 2
mood_like caring about others.csv : 820, mood_disgusted with self.csv : 2232
3052, 3032, 중복 : 20
mood_like caring about others.csv : 820, mood_like sharing with others.csv : 2396
3216, 2546, 중복 : 670
mood_like caring about others.csv : 820, mood_sleepy.csv : 301
1121, 1119, 

mood_wide awake.csv : 1126, mood_wild.csv : 7686
8812, 8775, 중복 : 37
mood_wide awake.csv : 1126, mood_intense.csv : 259
1385, 1382, 중복 : 3
mood_wide awake.csv : 1126, mood_calm.csv : 3645
4771, 4749, 중복 : 22
mood_wide awake.csv : 1126, mood_like having fun.csv : 4656
5782, 5769, 중복 : 13
mood_wide awake.csv : 1126, mood_full-of-pep.csv : 8942
10068, 10054, 중복 : 14
mood_wide awake.csv : 1126, mood_dissatisfied with self.csv : 2232
3358, 3351, 중복 : 7
mood_wide awake.csv : 1126, mood_clutched-up.csv : 8162
9288, 9237, 중복 : 51
mood_wide awake.csv : 1126, mood_content.csv : 301
1427, 1422, 중복 : 5
mood_wide awake.csv : 1126, mood_like caring about others.csv : 820
1946, 1939, 중복 : 7
mood_wide awake.csv : 1126, mood_hostile.csv : 9
1135, 1134, 중복 : 1
mood_wide awake.csv : 1126, mood_weary.csv : 9
1135, 1134, 중복 : 1
mood_wide awake.csv : 1126, mood_glad.csv : 28
1154, 1153, 중복 : 1
mood_wide awake.csv : 1126, mood_happy.csv : 6059
7185, 7153, 중복 : 32
mood_wide awake.csv : 1126, mood_serious.csv 

mood_glad.csv : 28, mood_like sharing with others.csv : 2396
2424, 2423, 중복 : 1
mood_glad.csv : 28, mood_clutched up.csv : 7767
7795, 7789, 중복 : 6
mood_glad.csv : 28, mood_angry at self.csv : 3008
3036, 3035, 중복 : 1
mood_glad.csv : 28, mood_cheerful.csv : 315
343, 340, 중복 : 3
mood_glad.csv : 28, mood_at rest.csv : 2814
2842, 2840, 중복 : 2
mood_glad.csv : 28, mood_down.csv : 3110
3138, 3137, 중복 : 1
mood_glad.csv : 28, mood_like being with others.csv : 2221
2249, 2248, 중복 : 1
mood_glad.csv : 28, mood_low.csv : 1293
1321, 1320, 중복 : 1
mood_glad.csv : 28, mood_wild.csv : 7686
7714, 7713, 중복 : 1
mood_glad.csv : 28, mood_calm.csv : 3645
3673, 3672, 중복 : 1
mood_glad.csv : 28, mood_like having fun.csv : 4656
4684, 4682, 중복 : 2
mood_glad.csv : 28, mood_dissatisfied with self.csv : 2232
2260, 2259, 중복 : 1
mood_glad.csv : 28, mood_clutched-up.csv : 8162
8190, 8182, 중복 : 8
mood_glad.csv : 28, mood_like caring about others.csv : 820
848, 847, 중복 : 1
mood_glad.csv : 28, mood_wide awake.csv : 1126
115

mood_serious.csv : 395, mood_like being alone.csv : 4604
4999, 4981, 중복 : 18
mood_serious.csv : 395, mood_bored.csv : 99
494, 489, 중복 : 5
mood_serious.csv : 395, mood_tired.csv : 328
723, 721, 중복 : 2
mood_serious.csv : 395, mood_strong.csv : 957
1352, 1340, 중복 : 12
mood_serious.csv : 395, mood_lonely.csv : 1751
2146, 2140, 중복 : 6
mood_serious.csv : 395, mood_angry.csv : 289
684, 669, 중복 : 15
mood_serious.csv : 395, mood_uncertain.csv : 19
414, 413, 중복 : 1
mood_serious.csv : 395, mood_disgusted with self.csv : 2232
2627, 2615, 중복 : 12
mood_serious.csv : 395, mood_like sharing with others.csv : 2396
2791, 2783, 중복 : 8
mood_serious.csv : 395, mood_sleepy.csv : 301
696, 694, 중복 : 2
mood_serious.csv : 395, mood_clutched up.csv : 7767
8162, 8151, 중복 : 11
mood_serious.csv : 395, mood_worried.csv : 28
423, 421, 중복 : 2
mood_serious.csv : 395, mood_angry at self.csv : 3008
3403, 3363, 중복 : 40
mood_serious.csv : 395, mood_cheerful.csv : 315
710, 709, 중복 : 1
mood_serious.csv : 395, mood_contemplat

mood_sad.csv : 2086, mood_happy.csv : 6059
8145, 7849, 중복 : 296
mood_sad.csv : 2086, mood_serious.csv : 395
2481, 2456, 중복 : 25
mood_sad.csv : 2086, mood_relaxed.csv : 345
2431, 2428, 중복 : 3
mood_sad.csv : 2086, mood_joyful.csv : 167
2253, 2252, 중복 : 1
mood_sad.csv : 2086, mood_quiet.csv : 1222
3308, 3291, 중복 : 17
mood_sad.csv : 2086, mood_carefree.csv : 73
2159, 2157, 중복 : 2
mood_sad.csv : 2086, mood_helpless.csv : 10
2096, 2094, 중복 : 2
mood_sad.csv : 2086, mood_mean.csv : 69
2155, 2153, 중복 : 2
mood_sad.csv : 2086, mood_alone.csv : 3595
5681, 5453, 중복 : 228
mood_sad.csv : 2086, mood_scared.csv : 85
2171, 2167, 중복 : 4
mood_sad.csv : 2086, mood_subdued.csv : 9
2095, 2094, 중복 : 1
mood_sad.csv : 2086, mood_upset.csv : 79
2165, 2131, 중복 : 34
mood_sad.csv : 2086, mood_shy.csv : 110
2196, 2190, 중복 : 6
mood_sad.csv : 2086, mood_mad.csv : 109
2195, 2185, 중복 : 10
mood_sad.csv : 2086, mood_stirred-up.csv : 8044
10130, 10080, 중복 : 50
mood_sad.csv : 2086, mood_self-centered.csv : 899
2985, 2968, 중

mood_joyful.csv : 167, mood_wild.csv : 7686
7853, 7849, 중복 : 4
mood_joyful.csv : 167, mood_intense.csv : 259
426, 425, 중복 : 1
mood_joyful.csv : 167, mood_calm.csv : 3645
3812, 3811, 중복 : 1
mood_joyful.csv : 167, mood_like having fun.csv : 4656
4823, 4794, 중복 : 29
mood_joyful.csv : 167, mood_full-of-pep.csv : 8942
9109, 9100, 중복 : 9
mood_joyful.csv : 167, mood_dissatisfied with self.csv : 2232
2399, 2398, 중복 : 1
mood_joyful.csv : 167, mood_excited.csv : 107
274, 266, 중복 : 8
mood_joyful.csv : 167, mood_clutched-up.csv : 8162
8329, 8316, 중복 : 13
mood_joyful.csv : 167, mood_content.csv : 301
468, 461, 중복 : 7
mood_joyful.csv : 167, mood_like caring about others.csv : 820
987, 986, 중복 : 1
mood_joyful.csv : 167, mood_overjoyed.csv : 1
168, 167, 중복 : 1
mood_joyful.csv : 167, mood_glad.csv : 28
195, 193, 중복 : 2
mood_joyful.csv : 167, mood_happy.csv : 6059
6226, 6124, 중복 : 102
mood_joyful.csv : 167, mood_sad.csv : 2086
2253, 2252, 중복 : 1
mood_joyful.csv : 167, mood_relaxed.csv : 345
512, 511, 중복

mood_attentive.csv : 14, mood_proud.csv : 143
157, 156, 중복 : 1
mood_attentive.csv : 14, mood_alert.csv : 87
101, 99, 중복 : 2
mood_satisfied.csv과 비교중 ...
mood_satisfied.csv : 13, mood_disgusted with self.csv : 2232
2245, 2244, 중복 : 1
mood_satisfied.csv : 13, mood_clutched up.csv : 7767
7780, 7777, 중복 : 3
mood_satisfied.csv : 13, mood_angry at self.csv : 3008
3021, 3020, 중복 : 1
mood_satisfied.csv : 13, mood_dissatisfied with self.csv : 2232
2245, 2244, 중복 : 1
mood_satisfied.csv : 13, mood_clutched-up.csv : 8162
8175, 8173, 중복 : 2
mood_satisfied.csv : 13, mood_content.csv : 301
314, 311, 중복 : 3
mood_satisfied.csv : 13, mood_happy.csv : 6059
6072, 6064, 중복 : 8
mood_satisfied.csv : 13, mood_relaxed.csv : 345
358, 357, 중복 : 1
mood_satisfied.csv : 13, mood_stirred-up.csv : 8044
8057, 8054, 중복 : 3
mood_satisfied.csv : 13, mood_self-centered.csv : 899
912, 911, 중복 : 1
mood_satisfied.csv : 13, mood_lazy.csv : 245
258, 257, 중복 : 1
mood_satisfied.csv : 13, mood_like being nice to others.csv : 3695


mood_mean.csv : 69, mood_wild.csv : 7686
7755, 7754, 중복 : 1
mood_mean.csv : 69, mood_intense.csv : 259
328, 327, 중복 : 1
mood_mean.csv : 69, mood_calm.csv : 3645
3714, 3713, 중복 : 1
mood_mean.csv : 69, mood_dissatisfied with self.csv : 2232
2301, 2299, 중복 : 2
mood_mean.csv : 69, mood_clutched-up.csv : 8162
8231, 8229, 중복 : 2
mood_mean.csv : 69, mood_content.csv : 301
370, 369, 중복 : 1
mood_mean.csv : 69, mood_annoyed.csv : 48
117, 116, 중복 : 1
mood_mean.csv : 69, mood_happy.csv : 6059
6128, 6125, 중복 : 3
mood_mean.csv : 69, mood_serious.csv : 395
464, 461, 중복 : 3
mood_mean.csv : 69, mood_sad.csv : 2086
2155, 2153, 중복 : 2
mood_mean.csv : 69, mood_quiet.csv : 1222
1291, 1290, 중복 : 1
mood_mean.csv : 69, mood_determined.csv : 27
96, 95, 중복 : 1
mood_mean.csv : 69, mood_alone.csv : 3595
3664, 3662, 중복 : 2
mood_mean.csv : 69, mood_scared.csv : 85
154, 153, 중복 : 1
mood_mean.csv : 69, mood_upset.csv : 79
148, 146, 중복 : 2
mood_mean.csv : 69, mood_shy.csv : 110
179, 178, 중복 : 1
mood_mean.csv : 69, moo

mood_delighted.csv : 1, mood_like having fun.csv : 4656
4657, 4656, 중복 : 1
mood_delighted.csv : 1, mood_like caring about others.csv : 820
821, 820, 중복 : 1
mood_delighted.csv : 1, mood_like being nice to others.csv : 3695
3696, 3695, 중복 : 1
mood_boastful_None.csv과 비교중 ...
mood_amazed.csv과 비교중 ...
mood_amazed.csv : 48, mood_like being alone.csv : 4604
4652, 4649, 중복 : 3
mood_amazed.csv : 48, mood_clutched up.csv : 7767
7815, 7813, 중복 : 2
mood_amazed.csv : 48, mood_shocked.csv : 11
59, 55, 중복 : 4
mood_amazed.csv : 48, mood_full of energy.csv : 9061
9109, 9107, 중복 : 2
mood_amazed.csv : 48, mood_playful.csv : 309
357, 355, 중복 : 2
mood_amazed.csv : 48, mood_astonished.csv : 1
49, 48, 중복 : 1
mood_amazed.csv : 48, mood_like everything is going wrong.csv : 1373
1421, 1420, 중복 : 1
mood_amazed.csv : 48, mood_afraid.csv : 22
70, 69, 중복 : 1
mood_amazed.csv : 48, mood_surprised.csv : 39
87, 80, 중복 : 7
mood_amazed.csv : 48, mood_wild.csv : 7686
7734, 7733, 중복 : 1
mood_amazed.csv : 48, mood_like havi

mood_upset.csv : 79, mood_clutched-up.csv : 8162
8241, 8238, 중복 : 3
mood_upset.csv : 79, mood_annoyed.csv : 48
127, 114, 중복 : 13
mood_upset.csv : 79, mood_glad.csv : 28
107, 106, 중복 : 1
mood_upset.csv : 79, mood_happy.csv : 6059
6138, 6134, 중복 : 4
mood_upset.csv : 79, mood_serious.csv : 395
474, 472, 중복 : 2
mood_upset.csv : 79, mood_sad.csv : 2086
2165, 2131, 중복 : 34
mood_upset.csv : 79, mood_quiet.csv : 1222
1301, 1299, 중복 : 2
mood_upset.csv : 79, mood_mean.csv : 69
148, 146, 중복 : 2
mood_upset.csv : 79, mood_alone.csv : 3595
3674, 3665, 중복 : 9
mood_upset.csv : 79, mood_amazed.csv : 48
127, 126, 중복 : 1
mood_upset.csv : 79, mood_scared.csv : 85
164, 163, 중복 : 1
mood_upset.csv : 79, mood_shy.csv : 110
189, 188, 중복 : 1
mood_upset.csv : 79, mood_mad.csv : 109
188, 181, 중복 : 7
mood_upset.csv : 79, mood_stirred-up.csv : 8044
8123, 8118, 중복 : 5
mood_upset.csv : 79, mood_rejected.csv : 5
84, 83, 중복 : 1
mood_upset.csv : 79, mood_engaged in thought.csv : 7026
7105, 7101, 중복 : 4
mood_upset.csv : 

mood_stirred-up.csv : 8044, mood_clutched up.csv : 7767
15811, 9490, 중복 : 6321
mood_stirred-up.csv : 8044, mood_ashamed.csv : 7
8051, 8050, 중복 : 1
mood_stirred-up.csv : 8044, mood_worried.csv : 28
8072, 8070, 중복 : 2
mood_stirred-up.csv : 8044, mood_angry at self.csv : 3008
11052, 10959, 중복 : 93
mood_stirred-up.csv : 8044, mood_cheerful.csv : 315
8359, 8345, 중복 : 14
mood_stirred-up.csv : 8044, mood_contemplative.csv : 52
8096, 8095, 중복 : 1
mood_stirred-up.csv : 8044, mood_full of energy.csv : 9061
17105, 16976, 중복 : 129
mood_stirred-up.csv : 8044, mood_hopeless.csv : 68
8112, 8109, 중복 : 3
mood_stirred-up.csv : 8044, mood_efficient.csv : 35
8079, 8078, 중복 : 1
mood_stirred-up.csv : 8044, mood_active.csv : 533
8577, 8558, 중복 : 19
mood_stirred-up.csv : 8044, mood_fearless.csv : 355
8399, 8394, 중복 : 5
mood_stirred-up.csv : 8044, mood_at rest.csv : 2814
10858, 10782, 중복 : 76
mood_stirred-up.csv : 8044, mood_playful.csv : 309
8353, 8334, 중복 : 19
mood_stirred-up.csv : 8044, mood_depressed.csv :

mood_self-centered.csv : 899, mood_healthy.csv : 2628
3527, 3508, 중복 : 19
mood_self-centered.csv : 899, mood_low.csv : 1293
2192, 2186, 중복 : 6
mood_self-centered.csv : 899, mood_frustrated.csv : 8
907, 906, 중복 : 1
mood_self-centered.csv : 899, mood_wild.csv : 7686
8585, 8582, 중복 : 3
mood_self-centered.csv : 899, mood_intense.csv : 259
1158, 1156, 중복 : 2
mood_self-centered.csv : 899, mood_calm.csv : 3645
4544, 4524, 중복 : 20
mood_self-centered.csv : 899, mood_like having fun.csv : 4656
5555, 5535, 중복 : 20
mood_self-centered.csv : 899, mood_full-of-pep.csv : 8942
9841, 9833, 중복 : 8
mood_self-centered.csv : 899, mood_dissatisfied with self.csv : 2232
3131, 2370, 중복 : 761
mood_self-centered.csv : 899, mood_anxious.csv : 46
945, 944, 중복 : 1
mood_self-centered.csv : 899, mood_clutched-up.csv : 8162
9061, 9034, 중복 : 27
mood_self-centered.csv : 899, mood_content.csv : 301
1200, 1196, 중복 : 4
mood_self-centered.csv : 899, mood_like caring about others.csv : 820
1719, 1709, 중복 : 10
mood_self-cente

mood_sorry.csv : 29, mood_clutched up.csv : 7767
7796, 7793, 중복 : 3
mood_sorry.csv : 29, mood_angry at self.csv : 3008
3037, 3035, 중복 : 2
mood_sorry.csv : 29, mood_at rest.csv : 2814
2843, 2842, 중복 : 1
mood_sorry.csv : 29, mood_like everything is going wrong.csv : 1373
1402, 1399, 중복 : 3
mood_sorry.csv : 29, mood_down.csv : 3110
3139, 3133, 중복 : 6
mood_sorry.csv : 29, mood_low.csv : 1293
1322, 1321, 중복 : 1
mood_sorry.csv : 29, mood_wild.csv : 7686
7715, 7714, 중복 : 1
mood_sorry.csv : 29, mood_clutched-up.csv : 8162
8191, 8189, 중복 : 2
mood_sorry.csv : 29, mood_content.csv : 301
330, 329, 중복 : 1
mood_sorry.csv : 29, mood_sad.csv : 2086
2115, 2109, 중복 : 6
mood_sorry.csv : 29, mood_relaxed.csv : 345
374, 373, 중복 : 1
mood_sorry.csv : 29, mood_alone.csv : 3595
3624, 3623, 중복 : 1
mood_sorry.csv : 29, mood_stirred-up.csv : 8044
8073, 8071, 중복 : 2
mood_sorry.csv : 29, mood_lazy.csv : 245
274, 273, 중복 : 1
mood_sorry.csv : 29, mood_sick.csv : 245
274, 271, 중복 : 3
mood_sorry.csv : 29, mood_at ease.

mood_engaged in thought.csv : 7026, mood_super.csv : 815
7841, 7831, 중복 : 10
mood_engaged in thought.csv : 7026, mood_sexy.csv : 936
7962, 7929, 중복 : 33
mood_engaged in thought.csv : 7026, mood_guilty.csv : 10
7036, 7034, 중복 : 2
mood_engaged in thought.csv : 7026, mood_proud.csv : 143
7169, 7168, 중복 : 1
mood_engaged in thought.csv : 7026, mood_alert.csv : 87
7113, 7110, 중복 : 3
mood_engaged in thought.csv : 7026, mood_nice.csv : 1265
8291, 8268, 중복 : 23
mood_loving.csv과 비교중 ...
mood_loving.csv : 205, mood_still.csv : 2316
2521, 2520, 중복 : 1
mood_loving.csv : 205, mood_like being alone.csv : 4604
4809, 4805, 중복 : 4
mood_loving.csv : 205, mood_disgusted with self.csv : 2232
2437, 2434, 중복 : 3
mood_loving.csv : 205, mood_like sharing with others.csv : 2396
2601, 2593, 중복 : 8
mood_loving.csv : 205, mood_clutched up.csv : 7767
7972, 7965, 중복 : 7
mood_loving.csv : 205, mood_angry at self.csv : 3008
3213, 3211, 중복 : 2
mood_loving.csv : 205, mood_cheerful.csv : 315
520, 519, 중복 : 1
mood_loving.

mood_friendly.csv : 318, mood_like being alone.csv : 4604
4922, 4917, 중복 : 5
mood_friendly.csv : 318, mood_comfortable.csv : 156
474, 473, 중복 : 1
mood_friendly.csv : 318, mood_strong.csv : 957
1275, 1272, 중복 : 3
mood_friendly.csv : 318, mood_lonely.csv : 1751
2069, 2066, 중복 : 3
mood_friendly.csv : 318, mood_disgusted with self.csv : 2232
2550, 2545, 중복 : 5
mood_friendly.csv : 318, mood_like sharing with others.csv : 2396
2714, 2705, 중복 : 9
mood_friendly.csv : 318, mood_clutched up.csv : 7767
8085, 8069, 중복 : 16
mood_friendly.csv : 318, mood_worried.csv : 28
346, 345, 중복 : 1
mood_friendly.csv : 318, mood_angry at self.csv : 3008
3326, 3322, 중복 : 4
mood_friendly.csv : 318, mood_cheerful.csv : 315
633, 621, 중복 : 12
mood_friendly.csv : 318, mood_full of energy.csv : 9061
9379, 9372, 중복 : 7
mood_friendly.csv : 318, mood_active.csv : 533
851, 850, 중복 : 1
mood_friendly.csv : 318, mood_at rest.csv : 2814
3132, 3128, 중복 : 4
mood_friendly.csv : 318, mood_playful.csv : 309
627, 604, 중복 : 23
mood_

mood_at ease.csv : 1799, mood_super.csv : 815
2614, 2610, 중복 : 4
mood_at ease.csv : 1799, mood_sexy.csv : 936
2735, 2731, 중복 : 4
mood_at ease.csv : 1799, mood_alert.csv : 87
1886, 1884, 중복 : 2
mood_at ease.csv : 1799, mood_nice.csv : 1265
3064, 3056, 중복 : 8
mood_egotistic_None.csv과 비교중 ...
mood_bad-tempered.csv과 비교중 ...
mood_bad-tempered.csv : 604, mood_unhappy.csv : 85
689, 687, 중복 : 2
mood_bad-tempered.csv : 604, mood_still.csv : 2316
2920, 2919, 중복 : 1
mood_bad-tempered.csv : 604, mood_like being alone.csv : 4604
5208, 5200, 중복 : 8
mood_bad-tempered.csv : 604, mood_bored.csv : 99
703, 698, 중복 : 5
mood_bad-tempered.csv : 604, mood_tired.csv : 328
932, 930, 중복 : 2
mood_bad-tempered.csv : 604, mood_strong.csv : 957
1561, 1554, 중복 : 7
mood_bad-tempered.csv : 604, mood_lonely.csv : 1751
2355, 2348, 중복 : 7
mood_bad-tempered.csv : 604, mood_angry.csv : 289
893, 886, 중복 : 7
mood_bad-tempered.csv : 604, mood_disgusted with self.csv : 2232
2836, 2831, 중복 : 5
mood_bad-tempered.csv : 604, mood_

mood_like being nice to others.csv : 3695, mood_wild.csv : 7686
11381, 11322, 중복 : 59
mood_like being nice to others.csv : 3695, mood_intense.csv : 259
3954, 3949, 중복 : 5
mood_like being nice to others.csv : 3695, mood_calm.csv : 3645
7340, 7271, 중복 : 69
mood_like being nice to others.csv : 3695, mood_like having fun.csv : 4656
8351, 7680, 중복 : 671
mood_like being nice to others.csv : 3695, mood_full-of-pep.csv : 8942
12637, 12587, 중복 : 50
mood_like being nice to others.csv : 3695, mood_dissatisfied with self.csv : 2232
5927, 5836, 중복 : 91
mood_like being nice to others.csv : 3695, mood_suspicious.csv : 12
3707, 3706, 중복 : 1
mood_like being nice to others.csv : 3695, mood_excited.csv : 107
3802, 3801, 중복 : 1
mood_like being nice to others.csv : 3695, mood_nervous.csv : 38
3733, 3731, 중복 : 2
mood_like being nice to others.csv : 3695, mood_clutched-up.csv : 8162
11857, 11762, 중복 : 95
mood_like being nice to others.csv : 3695, mood_content.csv : 301
3996, 3985, 중복 : 11
mood_like being nic

mood_super.csv : 815, mood_down.csv : 3110
3925, 3921, 중복 : 4
mood_super.csv : 815, mood_like being with others.csv : 2221
3036, 3026, 중복 : 10
mood_super.csv : 815, mood_healthy.csv : 2628
3443, 3438, 중복 : 5
mood_super.csv : 815, mood_low.csv : 1293
2108, 2084, 중복 : 24
mood_super.csv : 815, mood_wild.csv : 7686
8501, 8497, 중복 : 4
mood_super.csv : 815, mood_intense.csv : 259
1074, 1073, 중복 : 1
mood_super.csv : 815, mood_calm.csv : 3645
4460, 4457, 중복 : 3
mood_super.csv : 815, mood_like having fun.csv : 4656
5471, 5295, 중복 : 176
mood_super.csv : 815, mood_full-of-pep.csv : 8942
9757, 9705, 중복 : 52
mood_super.csv : 815, mood_dissatisfied with self.csv : 2232
3047, 3039, 중복 : 8
mood_super.csv : 815, mood_clutched-up.csv : 8162
8977, 8966, 중복 : 11
mood_super.csv : 815, mood_like caring about others.csv : 820
1635, 1634, 중복 : 1
mood_super.csv : 815, mood_wide awake.csv : 1126
1941, 1919, 중복 : 22
mood_super.csv : 815, mood_tense.csv : 10
825, 824, 중복 : 1
mood_super.csv : 815, mood_happy.csv :

mood_alert.csv : 87, mood_comfortable.csv : 156
243, 242, 중복 : 1
mood_alert.csv : 87, mood_strong.csv : 957
1044, 1041, 중복 : 3
mood_alert.csv : 87, mood_angry.csv : 289
376, 375, 중복 : 1
mood_alert.csv : 87, mood_quick.csv : 110
197, 196, 중복 : 1
mood_alert.csv : 87, mood_energetic.csv : 87
174, 173, 중복 : 1
mood_alert.csv : 87, mood_sleepy.csv : 301
388, 387, 중복 : 1
mood_alert.csv : 87, mood_clutched up.csv : 7767
7854, 7849, 중복 : 5
mood_alert.csv : 87, mood_angry at self.csv : 3008
3095, 3093, 중복 : 2
mood_alert.csv : 87, mood_active.csv : 533
620, 616, 중복 : 4
mood_alert.csv : 87, mood_at rest.csv : 2814
2901, 2897, 중복 : 4
mood_alert.csv : 87, mood_playful.csv : 309
396, 395, 중복 : 1
mood_alert.csv : 87, mood_down.csv : 3110
3197, 3193, 중복 : 4
mood_alert.csv : 87, mood_surprised.csv : 39
126, 123, 중복 : 3
mood_alert.csv : 87, mood_healthy.csv : 2628
2715, 2714, 중복 : 1
mood_alert.csv : 87, mood_wild.csv : 7686
7773, 7760, 중복 : 13
mood_alert.csv : 87, mood_calm.csv : 3645
3732, 3731, 중복 : 1


In [36]:
df1 = pd.read_csv('./mood_metadata/'+'mood_unhappy.csv')
df2 = pd.read_csv('./mood_metadata/'+'mood_unhappy.csv')

In [37]:
df_sum = pd.concat([df1, df2])

In [39]:
len(df2)

85

In [40]:
len(df_sum)

170

In [ ]:
mood_unhappy.csv : 85, mood_like being alone.csv : 4604
4689, 4674, 중복 : 15
    
mood_unhappy.csv : 85, mood_rejected.csv : 5
90, 88, 중복 : 2

mood_unhappy.csv : 85, mood_miserable.csv : 6
91, 89, 중복 : 2
    
mood_unhappy.csv : 85, mood_miserable.csv : 6
91, 89, 중복 : 2

In [130]:
metadata = pd.read_csv('metadata.csv', index_col='Unnamed: 0')

print(metadata.shape)
metadata.head()

(10426, 7)


color                                        description  \
7lAz-CrlED0  #F56C4D       Mixed media painting inspired by Stonehenge.   
5TK1F5VfdIk  #ECE5D0  Flowers in a Metal Vase.\r\nCreator: Abraham M...   
8E31onOcVXg  #E6C27C  Title: Wooded Landscape.\nCreator: Unknown.\nD...   
MvR30qxn-MM  #3A3A3C  Five Prints of Flowers in Glass Vases, created...   
yyzUYpEiJ6g  #EDF3F3  One of the most known pieces of street art in ...   

                                               alt_description  \
7lAz-CrlED0                              multicolored painting   
5TK1F5VfdIk                         red blue and white flowers   
8E31onOcVXg  green trees on green grass field under cloudy sky   
MvR30qxn-MM                       red and white floral textile   
yyzUYpEiJ6g                  bird organs and skeleton painting   

                                                         image  \
7lAz-CrlED0  https://images.unsplash.com/photo-154168067054...   
5TK1F5VfdIk  https://images.unsplash.com/photo-157978390261...   
8E31onOcVXg  https://images.unsplash.com/photo-158159248777...   
MvR30qxn-MM  https://images.unsplash.com/photo-157599587253...   
yyzUYpEiJ6g  https://images.unsplash.com/photo-153350256552...   

                                                link categories  \
7lAz-CrlED0  https://unsplash.com/photos/7lAz-CrlED0         []   
5TK1F5VfdIk  https://unsplash.com/photos/5TK1F5VfdIk         []   
8E31onOcVXg  https://unsplash.com/photos/8E31onOcVXg         []   
MvR30qxn-MM  https://unsplash.com/photos/MvR30qxn-MM         []   
yyzUYpEiJ6g  https://unsplash.com/photos/yyzUYpEiJ6g         []   

                                                          tags  
7lAz-CrlED0  [{'type': 'landing_page', 'title': 'art', 'sou...  
5TK1F5VfdIk  [{'type': 'landing_page', 'title': 'art', 'sou...  
8E31onOcVXg  [{'type': 'landing_page', 'title': 'art', 'sou...  
MvR30qxn-MM  [{'type': 'landing_page', 'title': 'art', 'sou...  
yyzUYpEiJ6g  [{'type': 'landing_page', 'title': 'art', 'sou...

In [10]:
for i in range(len(metadata)):
    print(i, end = ' ')
    urllib.request.urlretrieve(metadata.iloc[i]['image'], f"./image/{i}_{metadata.index[i]}.jpg")

0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 27

KeyboardInterrupt: 

# EDA

In [101]:
def clean_tags(tags):
    tags = ast.literal_eval(tags)
    #print(tags)
    results = [ i['title'] for i in tags ]
    return results

metadata['tags(clean)'] = metadata['tags'].apply(clean_tags)
metadata['tags(clean)']

7lAz-CrlED0           [art, modern art, painting]
5TK1F5VfdIk               [art, painting, flower]
8E31onOcVXg                [art, painting, human]
MvR30qxn-MM              [art, apparel, clothing]
yyzUYpEiJ6g               [art, katowice, poland]
                              ...                
Os657GJk9sQ        [art, graffiti, melbourne vic]
u7hgRgksNXU           [nature, outdoors, weather]
K-COGKqj0hM    [graffiti, building, architecture]
BtlHOw-390g               [nature, outdoors, sky]
_kk_joYTbGY            [graffiti, rapid city, sd]
Name: tags(clean), Length: 10426, dtype: object

In [105]:
tags_list = []
for tags in metadata['tags(clean)']:
    tags_list.extend(tags)
    
#tags_list

In [172]:
len(count)

1534

In [106]:
count={}
for i in tags_list:
    try: count[i] += 1
    except: count[i]=1
        
print(count)

{'art': 3593, 'modern art': 502, 'painting': 2859, 'flower': 42, 'human': 1583, 'apparel': 330, 'clothing': 228, 'katowice': 1, 'poland': 2, 'street': 11, 'automobile': 84, 'car': 71, 'bird': 66, 'animal': 182, 'symbol': 19, 'artwork': 4, 'wall': 451, 'religious': 1, 'Portrait': 6, 'military': 6, 'military uniform': 3, 'battle': 3, 'oil painting': 9, 'wood': 91, 'woman': 3, 'grey': 301, 'nude': 1, 'modernism': 1, 'graphics': 30, 'floral design': 9, 'watercolour': 7, 'building': 291, 'cottage': 4, 'house': 12, 'person': 1199, 'floral': 1, 'botanical': 1, 'plant': 953, 'gun': 1, 'framed works': 1, 'nature': 3022, 'ice': 48, 'hat': 9, 'face': 55, 'home decor': 56, 'text': 79, 'history': 9, 'machine': 43, 'wheel': 33, 'drawing': 28, 'pattern': 209, 'rug': 57, 'architecture': 190, 'furniture': 71, 'desk': 1, 'mountain': 667, 'cat': 3, 'kitten': 1, 'garden': 3, 'queen': 1, 'fish': 15, 'assembly line': 1, 'factory': 2, 'landscape': 211, 'profile': 1, 'transportation': 108, 'advertisement': 16

In [131]:
file = open('./english-word/english-adjectives.txt', mode='rt')
file

<_io.TextIOWrapper name='./english-word/english-adjectives.txt' mode='rt' encoding='UTF-8'>

In [132]:
strings = file.readlines()
#strings

In [134]:
adjectives = [i.replace('\n', '') for i in strings]

In [135]:
adjectives

['abandoned',
 'able',
 'absolute',
 'adorable',
 'adventurous',
 'academic',
 'acceptable',
 'acclaimed',
 'accomplished',
 'accurate',
 'aching',
 'acidic',
 'acrobatic',
 'active',
 'actual',
 'adept',
 'admirable',
 'admired',
 'adolescent',
 'adorable',
 'adored',
 'advanced',
 'afraid',
 'affectionate',
 'aged',
 'aggravating',
 'aggressive',
 'agile',
 'agitated',
 'agonizing',
 'agreeable',
 'ajar',
 'alarmed',
 'alarming',
 'alert',
 'alienated',
 'alive',
 'all',
 'altruistic',
 'amazing',
 'ambitious',
 'ample',
 'amused',
 'amusing',
 'anchored',
 'ancient',
 'angelic',
 'angry',
 'anguished',
 'animated',
 'annual',
 'another',
 'antique',
 'anxious',
 'any',
 'apprehensive',
 'appropriate',
 'apt',
 'arctic',
 'arid',
 'aromatic',
 'artistic',
 'ashamed',
 'assured',
 'astonishing',
 'athletic',
 'attached',
 'attentive',
 'attractive',
 'austere',
 'authentic',
 'authorized',
 'automatic',
 'avaricious',
 'average',
 'aware',
 'awesome',
 'awful',
 'awkward',
 'babyish',

In [123]:
unsplash_adjectives = []
for tag in tags_list:
    if tag in adjectives:
        unsplash_adjectives.append(tag)

In [124]:
unsplash_adjectives

['brown',
 'creative',
 'modern',
 'light',
 'light',
 'blue',
 'calm',
 'light',
 'black',
 'light',
 'light',
 'light',
 'urban',
 'light',
 'light',
 'light',
 'light',
 'urban',
 'light',
 'light',
 'brown',
 'red',
 'blue',
 'brown',
 'brown',
 'blue',
 'light',
 'light',
 'light',
 'concrete',
 'blue',
 'urban',
 'light',
 'light',
 'light',
 'black',
 'light',
 'light',
 'light',
 'light',
 'yellow',
 'red',
 'blue',
 'black',
 'colorful',
 'brown',
 'funny',
 'light',
 'light',
 'light',
 'light',
 'light',
 'urban',
 'light',
 'bronze',
 'urban',
 'black',
 'urban',
 'back',
 'white',
 'yellow',
 'female',
 'glass',
 'purple',
 'creative',
 'creative',
 'brown',
 'brown',
 'light',
 'light',
 'black',
 'brown',
 'light',
 'brown',
 'light',
 'light',
 'light',
 'light',
 'light',
 'brown',
 'brown',
 'blue',
 'light',
 'light',
 'light',
 'light',
 'light',
 'blue',
 'purple',
 'blue',
 'light',
 'light',
 'light',
 'light',
 'green',
 'blue',
 'light',
 'yellow',
 'brown',
 '

In [126]:
adjective_count = {}
for i in unsplash_adjectives:
    try: adjective_count[i] += 1
    except: adjective_count[i]=1
        
print(adjective_count)

{'brown': 44, 'creative': 5, 'modern': 1, 'light': 160, 'blue': 45, 'calm': 1, 'black': 14, 'urban': 17, 'red': 8, 'concrete': 1, 'yellow': 5, 'colorful': 1, 'funny': 3, 'bronze': 2, 'back': 1, 'white': 1, 'female': 2, 'glass': 3, 'purple': 13, 'green': 11, 'artistic': 1, 'pink': 5, 'political': 1, 'canine': 2}


In [145]:
import nltk
nltk.download()

showing info https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/index.xml


True

In [140]:
from nltk.corpus import wordnet as wn

In [146]:
for w in tags_list:
    tmp = wn.synsets(w)[0].pos()
    print(w, ":", tmp)

art : n


IndexError: list index out of range

In [149]:
wn.synsets(tags_list[0])[0

Synset('art.n.01')

In [153]:
pos_all = dict()
for w in tags_list:
    pos_l = set()
    for tmp in wn.synsets(w):
        if tmp.name().split('.')[0] == w:
            pos_l.add(tmp.pos())
    pos_all[w] = pos_l
unsplash_pos = pos_all

In [171]:
len(tags_list)

30150

{'brown': 44, 'creative': 5, 'modern': 1, 'light': 160, 'blue': 45, 'calm': 1, 'black': 14, 'urban': 17, 'red': 8, 'concrete': 1, 'yellow': 5, 'colorful': 1, 'funny': 3, 'bronze': 2, 'back': 1, 'white': 1, 'female': 2, 'glass': 3, 'purple': 13, 'green': 11, 'artistic': 1, 'pink': 5, 'political': 1, 'canine': 2}

In [177]:
adjective_count = []
for v, k in unsplash_pos.items():
    #print(v, k)
    if ('a' in k) or ('s' in k) or ('r' in k):
        adjective_count.append(v)
        
adjective_count

['human',
 'animal',
 'religious',
 'military',
 'grey',
 'floral',
 'brown',
 'accessory',
 'chicken',
 'minimal',
 'mural',
 'creative',
 'minimalist',
 'modern',
 'light',
 'abstract',
 'blue',
 'mosaic',
 'sunset',
 'calm',
 'cross',
 'black',
 'urban',
 'terminal',
 'downtown',
 'fly',
 'interior',
 'red',
 'velvet',
 'concrete',
 'yellow',
 'colorful',
 'funny',
 'dress',
 'smoking',
 'standing',
 'bronze',
 'invertebrate',
 'back',
 'white',
 'female',
 'native',
 'color',
 'purple',
 'historical',
 'rust',
 'green',
 'acid',
 'artistic',
 'sunrise',
 'satellite',
 'experimental',
 'pink',
 'political',
 'aquatic',
 'port',
 'pedestrian',
 'tropical',
 'beige',
 'mineral',
 'canine',
 'sitting',
 'in',
 'rose',
 'on']

In [173]:
len(adjective_count)

65

In [ ]:
adjective_count = {}
for i in unsplash_adjectives:
    try: adjective_count[i] += 1
    except: adjective_count[i]=1
        
print(adjective_count)

In [165]:
wn.synsets('amazing')

[Synset('amaze.v.01'),
 Synset('perplex.v.01'),
 Synset('amazing.s.01'),
 Synset('amazing.s.02')]

n: noun  
v: verb  
a: adjective  
s: adjective satellite  
antonym이 없는 단어를 의미한다. 특이한 경우인데, wn에서 adjective의 경우는 (adjective, verb, antonym)의 triple의 형태로 관리되는데, anotynym이 없을 경우만 따로 adjective satellite 로 관리한다.  
r: adverb  

In [178]:
adj_count = 0
for tag in tags_list:
    if tag in adjective_count:
        adj_count += 1
adj_count

3343